src/client/wallet/Transfer.js

```diff
@@ -119,6 +119,36 @@ export default class Transfer extends React.Component {
       return;
     }
 
+    if (value.length < 3) {
```

Comment:
probably could make both of these if statements into one seperate function above like `validateBalance`, but this one could be `validateUsernameLength`

Reply:
Also instead of having random numbers here, maybe instantiate them as variables like `minUsernameLength` & `maxUsernameLength`